stripe/build.gradle

```diff
@@ -6,12 +6,13 @@ configurations {
 
 dependencies {
     compile 'com.android.support:support-annotations:25.1.0'
+    compile 'com.android.support:support-v4:25.1.0'
```

Comment:
I probably should do that at least for the android support version stuff. I'll check on the build tools version variable -- the only reason I updated the support version is because Android Studio had an automated warning that I was out of date.